docs/application/resources.md

```diff
@@ -43,8 +43,41 @@ services:
       - gisdb
       - queue
       - cache
+```
+
+### EFS Resource (INSERT RELEASE VERSION HERE+)
```

Comment:
`INSERT RELEASE VERSION HERE+`

Reply:
I'm going to add the release version there when I release the convox/rack master